src/inetaddr.c

```diff
@@ -354,6 +354,25 @@ static struct inet_ifaddr *ifa_lookup(struct inet_device *idev,
     return NULL;
 }
 
+static struct inet_ifaddr *expired_ifa_lookup(struct inet_device *idev,
+                                              const union inet_addr *addr,
+                                              uint8_t plen, int af)
+{
+    struct inet_ifaddr *ifa;
+    lcoreid_t cid = rte_lcore_id();
+
+    list_for_each_entry(ifa, &ifa_expired_list[cid], h_list) {
+        if ((!plen || ifa->plen == plen) && (ifa->af == af)
+            && inet_addr_equal(af, &ifa->addr, addr)
+            && ifa->idev == idev) {
+            rte_atomic32_inc(&ifa->refcnt);
+            return ifa;
+        }
+    }
+
+    return NULL;
```

Comment:
`expired_ifa_lookup` almost does the same work as `inet_addr_ifa_get_expired`. Please combine them.

Reply:
already fix it.